framework/tlvf/AutoGenerated/include/tlvf/ieee_1905_1/tlvWscM1.h

```diff
@@ -88,6 +88,8 @@ class tlvWscM1 : public BaseClass
         WSC::sWscAttrDevicePasswordID& device_password_id_attr();
         WSC::sWscAttrConfigurationError& configuration_error_attr();
         WSC::sWscAttrOsVersion& os_version_attr();
+        bool is_vendor_ext_initialized() { return m_vendor_ext_init;}
```

Comment:
No need for a function, you can access these member variables from `isPostInitSucceeded()` in the derived class.